dvc/remote/ssh/connection.py

```diff
@@ -185,8 +185,17 @@ def download(self, src, dest, no_progress_bar=False, progress_title=None):
             self.sftp.get(src, dest, callback=pbar.update_to)
 
     def move(self, src, dst):
+        """Use `SFTP RENAME` operation when possible, otherwise,
+        fallback to copy and remove, since the former one doesn't
+        work across drives.
+        """
         self.makedirs(posixpath.dirname(dst))
-        self.sftp.rename(src, dst)
+
+        try:
+            self.sftp.rename(src, dst)
+        except OSError:
```

Comment:
would be great if we can detect a specific error

Reply:
@shcheklein , the exception that paramiko is throwing is not specific -- there's no `errno`, and the only message is "Failure", no other `args`. :disappointed:

Reply:
:(